site/content/docs/5.2/components/buttons.md

```diff
@@ -41,19 +41,19 @@ The recommended way of using an icon in a button is [an embedded SVG]({{< docsre
   <svg width="1rem" height="1rem" fill="currentColor" aria-hidden="true" focusable="false" class="me-1 overflow-visible">
     <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#success"/>
   </svg>
-  Primary
+  Small button
 </button>
 <button type="button" class="btn btn-primary">
   <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true" focusable="false" class="me-1">
     <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#success"/>
   </svg>
-  Primary
+  Medium button
```

Comment:
We never use "Medium" in terms of sizing in our docs. So we should stick with a simple "Button" here IMO:

```suggestion
  Button
```